gems/provider_amazon/provider_amazon.gemspec

```diff
@@ -0,0 +1,18 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+# Maintain your gem's version:
+require "provider_amazon/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "provider_amazon"
+  s.version     = ProviderAmazon::VERSION
+  s.authors     = ["Marcel Hild"]
+  s.email       = ["hild@b4mad.net"]
```

Comment:
Author and Email should probably include the ManageIQ Authors or whatever we have in the root of the project.  Not sure how this should be moving forward.